usb/esp_tinyusb/tusb_cdc_acm.c

```diff
@@ -382,26 +382,34 @@ esp_err_t tusb_cdc_acm_init(const tinyusb_config_cdcacm_t *cfg)
     acm->ringbuf_read_mux = xSemaphoreCreateMutex();
     if (acm->ringbuf_read_mux == NULL) {
         ESP_LOGE(TAG, "Creation of a ringbuf mutex failed");
-        free_obj(itf);
-        return ESP_ERR_NO_MEM;
+        ret = ESP_ERR_NO_MEM;
+        goto clean;
     }
```

Comment:
Nitpick: Could we use the `ESP_GOTO_ON_FALSE` macros here? to have the same style as in IDF code?

Reply:
done